src/localmodeling.jl

```diff
@@ -168,37 +212,43 @@ end
 
 
 """
-`TSP(tree,R,p,LocalModel,method,f)`
+    TSP(tree,R,q=R[end],p,LocalModel,method,f) -> s_pred
```

Comment:
First of all: **always** have 1 space between comma and next variable. In **every docstring and source code definition**.

We have to think a better name. 3-character long name is kind of non-sensical... Throw me suggestions!

`predict_timeseries` is very long. On the other hand, can we use a single, simple name like `predict` and put there methods? Then, this way, if/when we do the dynamical-networks-prediction-total-magic thing, we could still use the same syntax?

In any case, this can stay as is for now. But before we release the first version of the package, we need a good name for the highest level function.

Reply:
yeah, `predict` should be fine. Both networks-magic and spatio-temporal prediction seem _different_  enough -> there will probably be arguments to dispatch on.